packages/ui/react/src/icons/iconLoader.tsx

```diff
@@ -75,18 +75,24 @@ import { CircleBubble } from './CircleBubble'
 import { getImageAssetUrl } from './getImageAssetUrl'
 
 const bitbucketSVG = getImageAssetUrl('icons_svgs/bitbucket.svg')
-const bitbucketWhiteSVG = getImageAssetUrl('icons_svgs/bitbucket_white.svg')
-const c_sharpSVGsvg = getImageAssetUrl('icons_svgs/c_sharp.svg')
+const bitbucketWhiteSVG = getImageAssetUrl('icons_svgs/bitbucket-white.svg')
+const cSharpSVG = getImageAssetUrl('icons_svgs/c-sharp.svg')
 const dockerSVG = getImageAssetUrl('icons_svgs/docker.svg')
-const dotnetSVG = getImageAssetUrl('icons_svgs/dotnet.svg')
+const dotnetCoreSVG = getImageAssetUrl('icons_svgs/dotnetcore.svg')
+const dotnetCoreAspnetCoreSVG = getImageAssetUrl(
+  'icons_svgs/dotnet-aspnetcore.svg'
```

Comment:
The expected icon is suffixed with "core"

```suggestion
  'icons_svgs/dotnetcore-aspnetcore.svg'
```